supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -11,37 +11,38 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
-==== image:images/yes.png[yes] IBM eServer System p (noun)
-*Description*: Use "IBM eServer System p" for the first reference, and "IBM System p" or "System p" for subsequent references.
+[[ibm-power]]
+==== image:images/yes.png[yes] IBM Power® (noun)
+*Description*: _IBM Power_ represents the family of IBM Power processors (`ppc64le` architecture) and its products.
```

Comment:
`(`ppc64le` architecture)` is the only compatible architecture or is this just an example?